### DISCUSSION TITLE
Is it normal that installation validation takes too long?

### DISCUSSION THREAD

**Discussion by kjxbyz** [microsoft/winget-pkgs#159164] [Q&A]
Is it normal that installation validation takes too long?

[Azure Pipelines / WinGetSvc-Validation (WinGetSvc_Validation Installer Validation)](https://dev.azure.com/ms/winget-pkgs/_build/results?buildId=586344&view=results)

**Comment by Chill-Astro** [comment]
Actually yes. My 2 Packages themselves are taking multiple hours, due to the need of manual verification. It takes time, so patience is key.
